tests/integration/cli/config.go

```diff
@@ -6,8 +6,8 @@ const (
 	KeyringBackend = "test"
 	OutputFormat   = "json"
 	Gas            = "auto"
-	GasAdjustment  = "1.8"
-	GasPrices      = "50ncheq"
+	GasAdjustment  = "3.8"
+	GasPrices      = "90ncheq"
```

Comment:
Let's try to adjust back to the previous values and / or slightly increase if necessary, but not such unrealistic quantities.

e.g. Try with:

```suggestion
	GasAdjustment  = "1.8"
	GasPrices      = "50ncheq"
```

or 

```suggestion
	GasAdjustment  = "2.0"
	GasPrices      = "60ncheq"
```

Reply:
Ok @Eengineer1